.github/workflows/outdated-ak-version-issue.yml

```
@@ -0,0 +1,89 @@
+name: Check authentik version in new issue
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  check-version:
+    if: ${{ github.repository != 'goauthentik/authentik-internal' }}
```

```suggestion
    if: ${{ github.repository == 'goauthentik/authentik' }}
```
Would probably be more accurate?

---

Other workflows use !=. I can change this if you want?

---

yeah let's change it here and update the other ones separately